src/components/DashboardMain.js

```diff
@@ -44,230 +44,80 @@ class Main extends Component {
-                </ul>
-              </nav>
-            </div>
-            {routesElem("")}
-            {profElem}
-          </div>
-        ];
-      } else if (size === "xs") {
-        contentElem = [
-          <div className="profile-head">
-            <div className="row">
-              <div className="col-xs-4 profile-head">
-                <h3>{this.props.l10n("main.profile_title")}</h3>
-              </div>
-              <div className="col-xs-1" />
-              <div className="col-xs-7 profile-head">
-                <NotificationsContainer />
-              </div>
-            </div>
-            <div className="row">
-              <div className="col-xs-1" />
-              <div className="col-xs-11 pending-actions-container">
-                <PendingActionsContainer history={history} />
-              </div>
-            </div>
-          </div>,
-          <div className="row">
-            <div id="col-xs-12 profile-navbar-xs">
-              <nav
-                id="profile-menu-small"
-                className="navbar navbar-light"
-                role="navigation"
-              >
-                <a className="navbar-brand" href="#">
-                  {this.props.l10n("main.menu")}
-                </a>
-                <button
-                  className="navbar-toggler"
-                  type="button"
-                  onClick={() =>
-                    this.setState({ openTabs: !this.state.openTabs })
-                  }
-                >
-                  <span className="navbar-toggler-icon" />
-                </button>
-              </nav>
-            </div>
-            <Collapse
-              id="eduid-menu-items-xs"
-              isOpen={this.state.openTabs}
-              className="text-center"
-            >
-              <nav className="navbar navbar-light bg-light">
-                <ul className="nav nav-stacked nav-tabs navbar-nav">
-                  {tabsElems("main-nav-tabs tab-link btn-block")}
-                </ul>
-              </nav>
-            </Collapse>
-            {routesElem("")}
-            {profElem}
-          </div>
-        ];
-      }
-    } else {
-      contentElem = routesElem(<NotificationsContainer />);
-    }
-
     return (
       <FetchingContext.Provider value={this.state}>
         <SplashContainer />
         <div className="container-fluid">
           <HeaderContainer />
           <ConnectedRouter history={history}>
             <div id="content-block">
-              <div
-                className="tabbable well profile-combo"
-                id="profile-content-area"
+              <Link
+                className="button"
+                id="submit-button-link"
+                to={`/profile/settings/`}
               >
-                {contentElem}
+                <button
+                  id="settings-button"
+                  type="submit"
+                >
+                  Settings
```

Comment:
I'm not sure it's a good idea to add non-internationalized messages. If you have a plan about it, to keep track of all translatable messages and collect them into a catalog etc, I'm fine :)
If you don't have a plan, or your plan is just to worry later, it can be more time consuming and error prone than it seems ;-)

Reply:
Good point! And sorry if I didn't mention that I was planning for all content to be static placeholder until we have a rough idea of what we want things to say. I hope it will be ok to keep on working on static versions until we have agreed what all text should be, but let me know if you have used a better way before.

Reply:
ok agreed